libsolidity/lsp/LanguageServer.cpp

```diff
@@ -427,44 +430,47 @@ void LanguageServer::handleInitialize(MessageID _id, Json::Value const& _args)
 	m_client.reply(_id, std::move(replyArgs));
 }
 
-void LanguageServer::handleInitialized(MessageID, Json::Value const&)
+void LanguageServer::handleInitialized(MessageID, Json const&)
 {
 	if (m_fileLoadStrategy == FileLoadStrategy::ProjectDirectory)
 		compileAndUpdateDiagnostics();
 }
 
-void LanguageServer::semanticTokensFull(MessageID _id, Json::Value const& _args)
+void LanguageServer::semanticTokensFull(MessageID _id, Json const& _args)
 {
-	auto uri = _args["textDocument"]["uri"];
+	if (_args.contains("textDocument") && _args["textDocument"].contains("uri"))
```

Comment:
I wonder if anything should happen in the ``else`` cases for these... like an error reply or so... but then again, maybe we shouldn't care too much, since this is scheduled for removal anyways...

Reply:
I think I would just add `m_client.error(_id, ErrorCode::InvalidParams, "Invalid parameter: textDocument.uri expected.");` as the `else`case. And will try to just do this for everything where I see this might happen.

Reply:
ok.. looks like this is the only case where this might be happen